app/models/manager_refresh/inventory_object_lazy.rb

```diff
@@ -36,7 +36,7 @@ def transitive_dependency?
       # and a :stack is a relation to another object, in the InventoryObject object,
       # then this relation is considered transitive.
       !!(key && (inventory_collection.dependency_attributes.keys.include?(key) ||
```

Comment:
fixed, but I still like the key && ... there, since it speeds up the condition :-)

also using key? instead of include?